perception/autoware_tensorrt_bevdet/include/autoware/tensorrt_bevdet/bevdet.hpp

```diff
@@ -0,0 +1,241 @@
+// Copyright 2024 AutoCore, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+#ifndef AUTOWARE__TENSORRT_BEVDET__BEVDET_HPP_
+#define AUTOWARE__TENSORRT_BEVDET__BEVDET_HPP_
+
+#include "NvInfer.h"
+#include "common.hpp"
+#include "data.hpp"
+#include "postprocess.hpp"
+
+#include <Eigen/Core>
+#include <Eigen/Geometry>
+
+#include <NvOnnxParser.h>
+#include <yaml-cpp/yaml.h>
+
+#include <algorithm>
+#include <map>
+#include <memory>
+#include <string>
+#include <vector>
+
+namespace autoware::tensorrt_bevdet
+{
+class adjFrame
```

Comment:
I recommend you to use UpperCamelCase for the class name
```suggestion
class AdjFrame
```

FYI https://google.github.io/styleguide/cppguide.html#Naming